packages/kit/types/endpoint.d.ts

```diff
@@ -1,16 +1,18 @@
 import { ServerRequest } from './hooks';
 import { Headers, MaybePromise } from './helper';
 
-type JSONValue =
+type JSONValue = string | number | boolean | null | JSONValue[] | { [key: string]: JSONValue };
+
+type JSONBody =
```

Comment:
I might call it `JSONResponse` instead to clarify that it's only a response body and not a request body

Reply:
ok